tools/stability-checker/README.md

```diff
@@ -1,37 +1,55 @@
 # Stability Checker
 
 ## Overview
-
-The Stability Checker runs the `testing-kyma.sh` script in a loop, and reports the results on a Slack channel.
+The purpose of the Stability Checker is to check if a cluster is stable. 
+To ensure the cluster's stability, install the Stability Checker in the cluster. 
+It runs a testing script in a loop and sends notifications with the test executions summary on a Slack channel.
 
 ## Installation
+Install the Stability Checker on the Kyma cluster as a Helm chart. Find the chart definition in the `deploy/chart` directory.
+1. Configure kubectl and helm to point your cluster by specifying **KUBECONFIG** environment variable. 
```

Comment:
`to point to your cluster by specifying the **KUBECONFIG** environment variable. `